site/src/component/ChangelogModal/ChangelogModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import './ChangelogModal.scss';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Row from 'react-bootstrap/Row';
+
+const DESCRIPTION = 'You can now view recently added features to the PeterPortal website, listed in this modal.';
+const IMAGE_URL =
+  'https://media.tenor.com/ufm_0t3ACEAAAAAM/ginger-cat-ginger-cat-eating-then-staring-at-the-camera.gif';
+
+const ChangelogModal = () => {
+  const [showModal, setShowModal] = useState(false);
+
+  const currentMonth = new Date().toLocaleString('default', { month: 'long', year: 'numeric' });
```

Comment:
We should replace this with an additional constant above for the date (rather than just the month) of the last changelog update. This will then be manually updated anytime the description and image are updated. That way, this can support multiple updates to the changelog within a month.